src/components/shared/Chart.css

```diff
@@ -1,32 +1,37 @@
 /*We can use scoped css for each component but the setting were almost the same everytime*/
+body {
+  background-color: black;
+}
+
 .chart {
-    float: left;
-    width: 45%;
-    height: 100%;
-    background-color: black;
-    border: 1px solid rgba(255,255,255,0.3); /* white with a bit of opacity */
+  float: left;
+  width: 45%;
+  height: 100%;
+  background-color: black;
+  border: 1px solid rgba(255, 255, 255, 0.3); /* white with a bit of opacity */
 }
 
 .chart_container {
-    height: 600px;
-    display: flex;
-    justify-content: space-around;
-    margin-top: 2%;
+  height: 600px;
+  justify-content: space-around;
+  margin-top: 2%;
 }
 
 .spinner {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
 }
 
-body {
-    background-color: black;
+.title {
+  text-align: center;
+  color: #fff;
+  margin-top: 40px;
 }
 
-.title {
-    text-align: center;
-    color: #fff;
-    margin-top: 40px;
+@media (min-width: 768px) {
```

Comment:
I am no .css guy, but what does this improvement add ? 

Having the window at > 768 pixels the chart still looks bad with this fix or not. 
Maybe there is an example I am missing.